connectors/connector-xero/connector_xero/commands/createInvoice.py

```diff
@@ -199,6 +199,7 @@ def store_xero_oauth2_token(token):
             quantity=1.0,
             unit_amount=self.amount,
             account_code="400",
+            tax_type="NONE",
```

Comment:
lol? it's a good thing they require this parameter.

Reply:
heh if only more places allowed setting it...

really its going to have to be configured based on how the xero account/billing etc is set up. but can tweak that after the flow is working on demo.